pkg/config/config.go

```diff
@@ -459,6 +459,8 @@ type Update struct {
 	GitHubMonitor *GitHubMonitor `json:"github,omitempty" yaml:"github,omitempty"`
 	// The configuration block for transforming the `package.version` into an APK version
 	VersionTransform []VersionTransform `json:"version-transform,omitempty" yaml:"version-transform,omitempty"`
+	// Prefix filter to apply when searching releases
+	FilterPrefix string `json:"filter-prefix,omitempty" yaml:"filter-prefix,omitempty"`
```

Comment:
I wonder if putting it under the `update` block will cause confusion with the existing filter-prefix that is under the `github` block?  I.e. would it be better if we move this under the `release-monitor` block?

Reply:
Or maybe Deprecate the 'TagFilterPrefix' in github block?  filtering strings isn't really "provider specific".

I'm happy to do whatever you think is right.